frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/document_cover_widget.dart

```diff
@@ -44,6 +46,7 @@ class DocumentHeaderBlockKeys {
   static const String coverType = 'cover_selection_type';
   static const String coverDetails = 'cover_selection';
   static const String icon = 'selected_icon';
+  static const String align = 'cover_selection_align';
```

Comment:
Using name `offset` is more reasonable than `align`. Also, please add the JSON string to explain what the value will be included.